### DISCUSSION TITLE
feat: hope we can modify the tags

### DISCUSSION THREAD

**Discussion by maru7777** [open-webui/open-webui#7173] [General]
now tags can only generated by AI, we  can't control it , you must select it in a drop menu,which is correlated with tags table 

can we  have our own tags ? for example,  when we try to add a new one we can input something ,instead of select onw from the drop menu

**Comment by tkg61** [comment]
I am on 0.4.2 and am able to add my own tags, maybe a screenshot or something would help?

**Comment by rajivraghu** [comment]
![image](https://github.com/user-attachments/assets/7173936d-f867-4f70-a613-ce1172be2ddf)
I am unable to see  add tag... i see that if we use tags, it will internally add as metadata in chromadb for filtering data.
I'm on the latest openwebui docker 
@tkg61

**Reply by tkg61** [reply]
Ah I see, when you said tags I thought you meant chats and tagging those.
